packages/neo-one-node-bin/src/utils/getConfiguration.ts

```diff
@@ -0,0 +1,96 @@
+import { FullNodeEnvironment, FullNodeOptions as ConfigOptions } from '@neo-one/node';
+import { createTest } from '@neo-one/node-neo-settings';
+import chokidar from 'chokidar';
+import envPaths from 'env-paths';
+import fs from 'fs-extra';
+import rc from 'rc';
+import { BehaviorSubject, concat, Observable, of } from 'rxjs';
+import { scan } from 'rxjs/operators';
+import SeamlessImmutable from 'seamless-immutable';
+
+interface ConfigSettings {
+  readonly decrementInterval?: number;
+  readonly generationAmount?: ReadonlyArray<number>;
+  readonly messageMagic?: number;
+  readonly addressVersion?: number;
+  readonly privateKeyVersion?: number;
+  readonly vm?: {
+    readonly storageContext: {
+      readonly v0: {
+        readonly index: number;
+      };
+    };
+  };
+  readonly secondsPerBlock?: number;
+  readonly maxTransactionsPerBlock?: number;
+  readonly memPoolSize?: number;
+  readonly features?: {
+    readonly structClone: number;
+  };
+}
+
+const getFullSettings = (settings: ConfigSettings) => ({
+  ...createTest(),
```

Comment:
Same comment from first review:

Not quite - don't merge some arbitrary `settings` object into the settings of `createTest`. Instead you should do something like
```
export interface NodeBinEnvironment {
  settings: SettingsEnvironment;
  fullNode: FullNodeEnvironment; 
}
```
then choose which of `createMain` or `createTest` to use based on a property in `SettingsEnvironment` and pass arguments from the properties of `SettingsEnvironment` to the function.